crypto/keys/mintkey/mintkey.go

```diff
@@ -83,6 +83,10 @@ func UnarmorInfoBytes(armorStr string) ([]byte, error) {
 // UnarmorPubKeyBytes returns the pubkey byte slice, a string of the algo type, and an error
 func UnarmorPubKeyBytes(armorStr string) (bz []byte, algo string, err error) {
 	bz, header, err := unarmorBytes(armorStr, blockTypePubKey)
+	if err != nil {
```

Comment:
we should enable the linter `errcheck` it would have caught this.

Reply:
Totally!